boilerplates/rshiny/README.md

```diff
@@ -0,0 +1,26 @@
+# Rshiny Application
+## Purpose of this quickstarter
+Provisions a Rshiny application within openshift using crowd authentification.
+
+## What files / architecture is generated?
++-- Jenkinsfile  
++-- Dockerfile    
++-- app.R
+
+## Frameworks used
+R/Rshiny
+
+## Usage - how do you start after you provisioned this quickstarter
+The quickstarter sets up two pods in openshift. The rshiny application is routed through a crowd authentication proxy.
```

Comment:
please link to the image on github in ods-core/Shared images

Reply:
Done in newest commit